[metofficedatahub] 2025 API Key adjustments (#19627)
openhab/openhab-addons: bundles/org.openhab.binding.metofficedatahub/README.md

The bridge counts the total number of requests from 00:00 -> 23:59 under its properties during the runtime of the system.
(This reset's if OH restarts, or the binding resets).

| Name                 | Type    | Description                                                                               | Default Values |
|----------------------|---------|-------------------------------------------------------------------------------------------|----------------|
| siteRateDailyLimit   | Number  | This is a daily poll limit for the SiteSpecific API, while the Thing ID remains the same. | 250            |
| siteApiKey           | String  | The API Key for the Site Specific subscription in your MET Office Data Hub account.       |                |

> | disableKeyValidation | Boolean | If enabled the binding will not attempt to validate a valid key format has been entered.  | false          |

Reviewer: Maybe change it to be more readable. 

```suggestion
| disableKeyValidation | Boolean | If enabled, the binding will skip validation of the key format.  | false          |
```

Author: yeah I like that better will update that in the readme and the description of the config item.

Reviewer: Oh, i did not properly align the whitespaces. 
(and properties file)

Author: Thank you @lsiepel - also sorry is it the 5.0.X branch for backporting. Are we good to back port the new setting as well?

Author: ah bleep that's a silly mistake to forget thanks running now

Author: just pushed thanks :) (Need to get that one added to copilots auto-checks :)
